cmake/FindWayland.cmake

```diff
@@ -0,0 +1,17 @@
+find_package(PkgConfig)
+
+if(PKG_CONFIG_FOUND)
+    pkg_search_module(WAYLAND_CLIENT IMPORTED_TARGET wayland-client)
+endif()
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(Wayland
+    REQUIRED_VARS
+        WAYLAND_CLIENT_LIBRARIES
+        WAYLAND_CLIENT_INCLUDE_DIRS
```

Comment:
The Ubuntu job is likely failing because `wayland-client.h` is installed in the default include dir, and no `-I` flag is thus returned by pkg-config/pkgconf; as no flags are needed, `WAYLAND_CLIENT_INCLUDE_DIRS` is empty, and `find_package_handle_standard_args()` returns an error. You could simply remove `WAYLAND_CLIENT_INCLUDE_DIRS`, everything is contained in the imported target anyway.